server/core/src/main/java/org/infinispan/server/core/ServerCoreBlockHoundIntegration.java

```diff
@@ -0,0 +1,51 @@
+package org.infinispan.server.core;
+
+import org.infinispan.commons.internal.CommonsBlockHoundIntegration;
+import org.kohsuke.MetaInfServices;
+
+import io.netty.util.concurrent.GlobalEventExecutor;
+import reactor.blockhound.BlockHound;
+import reactor.blockhound.integration.BlockHoundIntegration;
+
+@MetaInfServices
+public class ServerCoreBlockHoundIntegration implements BlockHoundIntegration {
+   @Override
+   public void applyTo(BlockHound.Builder builder) {
+      builder.allowBlockingCallsInside(GlobalEventExecutor.class.getName(), "addTask");
+      builder.allowBlockingCallsInside(GlobalEventExecutor.class.getName(), "takeTask");
+
+      // The xerces parser when it finds a parsing error will print to possibly a file output - ignore
+      builder.allowBlockingCallsInside("com.sun.org.apache.xerces.internal.util.DefaultErrorHandler", "printError");
```

Comment:
Does this actually come up in our test suite? Can we replace the error handler, or is it used internally by something else, like Nashorn?

Reply:
We have tests in REST that have parsing errors at runtime and the parser was either writing to Stderror or trying to write to a file or something. I wasn't too picky about looking into tbh.